index.html

```diff
@@ -6,127 +6,265 @@
+              <a
+                href="Amanda_Kube_CV_111423.pdf"
+                target="_blank"
+                style="font-size: 20px"
+                >CV</a
+              >
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
 
-		<!-- Collect the nav links, forms, and other content for toggling -->
-		<div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
-		  <ul class="nav navbar-nav">
-				  <li><a href="index.html" style="font-size: 20px;">Home</a></li>
-			  	<li><a href="teaching.html" style="font-size: 20px;">Teaching</a></li>
-			  	<li><a href="publications.html" style="font-size: 20px;">Publications</a></li>
-				  <li><a href="Amanda_Kube_CV_111423.pdf" target="_blank" style="font-size: 20px;">CV</a></li> 
-		  </ul>
-		</div>
-	  </div>
-	</nav>
-  
     <div class="container">
-
-        <div class="row">
-            <!-- Contact Info on the Sidebar -->
-            <div class="col-md-4">
-		    <div style="font-size:16px; margin-left:-7vh"">
-                <img class="img-responsive" src="headshot_jotte.jpg" alt="image of me" style="width:280px;height:225px;"><br>
-                <div style="font-family: 'Oswald', sans-serif; font-size: 30px;"><b>Amanda R. Kube Jotte, M.S., Ph.D.</b></div><br>
-		<div style="font-family: 'Oswald', sans-serif; font-size: 23px;"><b>Assistant Instructional Professor</b></div><br>
-                <p><b>akube@uchicago.edu</b><br>
-                <p>Data Science Institute<br>
-                The University of Chicago<br>
-                </p>
-			    <a href="https://twitter.com/AmandaRKube"
-                                        title=""><img alt="twitter" target="_blank"
-                                            src="twitter.png" style="width:32px;height:32px;"></a>
-		   
-		    <a href="https://www.linkedin.com/in/amanda-kube-488071a8/"
-                                        title=""><img alt="linkedin" target="_blank"
-                                            src="linkedin.png" style="width:32px;height:32px;"></a>
-					
-		    <a href="https://scholar.google.com/citations?user=eML9qi8AAAAJ&hl=en&inst=2230987035966559800"
-                                        title=""><img alt="Google Scholar" target="_blank"
-                                            src="google-scholar.svg" style="width:32px;height:32px;"></a>
-					
-		    <a href="https://www.researchgate.net/profile/Amanda_Kube?cp=shp"
-                                        title=""><img alt="Follow me on ResearchGate" target="_blank"
-                                            src="researchgate.svg" style="width:32px;height:32px;"></a>
-					
-		    <a href="https://orcid.org/0000-0002-4814-3209"
-                                        title=""><img alt="orcid" target="_blank"
-							   src="orcid.svg" style="width:32px;height:32px;"></a>
+      <div class="row">
+        <!-- Contact Info on the Sidebar -->
+        <div class="col-sm-12 col-md-4">
```

Comment:
updating to say "take up 12 columns (full width) at mobile but only 4/12 columns on md and up sizes"